### PR TITLE
Group

### PR DESCRIPTION
**Pull request by Enriquesuarezgtte**
Running testNG suite from testng2.xml

In groupSmokeSuite.xml (before testng2.xml) I added a test filtering and running only methods from classes "test.Basics" and  "test.LoginTest" marked with group called "Smoke".

imagen

### CODE REVIEW

File: src/test/Basics.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void afterMethod() {
     System.out.println("After Method");
   }
 
-  @Test
+  @Test(groups= {"Smoke"} )
```
Comment (yesid-lopez) [inline src/test/Basics.java]: @Test(groups= {"Smoke"})

File: src/test/LoginTest.java [hunk 2]
```diff
@@ -4,12 +4,13 @@
 
 public class LoginTest {
 	
-	@Test
-	public void webLoginCarLoan() {
+	@Test(groups= {"Smoke"}, timeOut = 400)
+	public void webLoginCarLoan() throws InterruptedException {
+	  Thread.sleep(350);
 		System.out.println("weblogin");
 	}
```
Comment (yesid-lopez) [inline src/test/LoginTest.java]: indent code

**Comment by yesid-lopez** [comment]
It's very helpful that you could add an explanation about the testng2.xml

File: testng2.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yesid-lopez) [inline testng2.xml]: could you create a better file name than testng2?

File: src/test/LoginTest.java (hunk 2 above)
Comment (Enriquesuarezgtte) [reply src/test/LoginTest.java]: I was confused to read this becasuse on Eclipse appeared indented, then I openned same file on VSCode and it wasn't.

**Comment by Enriquesuarezgtte** [comment]
> 
> 
> It's very helpful that you could add an explanation about the testng2.xml

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**